src/ResourceManager/Network/Commands.Network/ApplicationGateway/FirewallDisabledRuleGroup/NewAzureApplicationGatewayFirewallDisabledRuleGroupConfigCommand.cs

```diff
@@ -0,0 +1,30 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Network.Models;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.New, "AzureRmApplicationGatewayFirewallDisabledRuleGroupConfig"), 
```

Comment:
As discussed offline, there is no shorter name that still matches our current pattern. We are actually thinking of introducing shorter aliases for all of the cmdlets (agw instead of applicationGateway?), this will not be part of this PR though.